number_theory/modular_unittest.cpp

```diff
@@ -0,0 +1,41 @@
+#include <gtest/gtest.h>
+
+#include "number_theory/modular.h"
+
+namespace tql {
```

Comment:
The namespace should be `tql::number_theory`.

Reply:
I add a line `namespace number_theory` to make it consistent with the current code base.